src/website/app/demos/Button/index.js

```diff
@@ -26,6 +26,7 @@ export default {
   doc,
   examples,
   slug: 'button',
+  syntax: `import Button from 'mineral-ui/Button'`,
```

Comment:
The snippets should probably end with a `;`.

Reply:
I was thinking we shouldn't have an opinion in the Great Semicolon Debate, and I did a non-scientific sample of npm READMEs, and it seemed like a mixture. I guess our codebase has `;`, so I'll update the examples.